notebooks/Grid Search for Binary Classification.ipynb

```diff
@@ -208,8 +208,9 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "lagrange_multipliers = [x.lagrange_multiplier for x in (first_sweep.all_results)]\n",
-    "lagrange_multipliers"
+    "lagrange_multipliers = [x.lagrange_multiplier.iat[0] - x.lagrange_multiplier.iat[2] for x in (first_sweep.all_results)]\n",
+    "lagrange_multipliers\n",
+    "first_sweep.best_result.lagrange_multiplier#[(\"+\", \"all\", 2)]"
```

Comment:
This is going to need some explanation, and I also need to finish updating this notebook for the new Lagrange multiplier style